src/test/java/uk/gov/companieshouse/fixtures/PaymentFixtures.java

```diff
@@ -33,4 +33,12 @@ public static PaymentGetResponse generatePaymentGetResponse(String eTag, String
 
         return response;
     }
+
+    public static PaymentPatchRequest generatePaymentPatchRequest() {
+        return new PaymentPatchRequest() {{
```

Comment:
I tried creating fixtures like this originally and it started bolting on additional fields on the Object and causing strange behaviour when inserting into MongoDB...